src/Models/Environment.php

```diff
@@ -226,20 +226,27 @@ public function connectionInfo()
      */
     public function cacheserverConnectionInfo()
     {
-        $cacheServerBindings = $this->getBindings()->getByType('cacheserver');
-        if (empty($cacheServerBindings)) {
-            return [];
+        $env_vars = $this->fetchEnvironmentVars();
+        if (property_exists($env_vars, 'CACHE_PORT')) {
```

Comment:
vars assignments could be optimized into this:
```php
$port = $env_vars['CACHE_PORT'] ?? null;
```
(please see the comment on `fetchEnvironmentVars()`)